inc/targetchange.class.php

```diff
@@ -960,11 +962,17 @@ public function save(PluginFormcreatorForm_Answer $formanswer) {
          'checklistcontent'
       ];
       foreach ($changeFields as $changeField) {
-         $data[$changeField] = $this->fields[$changeField];
+         //TODO: 2.7.0 rename PluginFormcreatorTargetChange's comment into content
+         if ($changeField == 'content') {
+            // This handles mismatch of the column content in Change itemtype and comment in TargetChange itemtype
+            $data[$changeField] = $this->fields[$changeField];
+         } else {
+            $data[$changeField] = $this->fields['comment'];
+         }
```

Comment:
column names should be the same between Change and PluginFormcreatorTargetChange, but currently it is not. 

Update the schema planned for 2.7.0